corehq/apps/domain/deletion.py

```diff
@@ -166,25 +162,33 @@ def _terminate_subscriptions(domain_name):
 
 def delete_all_cases(domain_name):
     logger.info('Deleting cases...')
-    case_ids = CommCareCase.objects.get_case_ids_in_domain(domain_name)
-    case_ids += CommCareCase.objects.get_deleted_case_ids_in_domain(domain_name)
-    for case_id_chunk in chunked(with_progress_bar(case_ids, stream=silence_during_tests()), 500):
```

Comment:
I didn't notice this before, but apparently this had used `with_progress_bar` (with `silence_during_tests`). Maybe that should be added back?